diff --git a/setup.py b/setup.py
index be1a87e..bab2ee3 100644
--- a/setup.py
+++ b/setup.py
@@ -62,13 +62,6 @@ setup(
 	cmdclass = cmdclass,
 	setup_requires = setup_requires,
 	install_requires = ['six'],
-	ext_modules = [
-		Extension(
-			'ligo.segments.__segments',
-			['src/segments.c', 'src/infinity.c', 'src/segment.c', 'src/segmentlist.c'],
-			include_dirs = ['src'],
-		),
-	],
 	classifiers = [
 		'Development Status :: 5 - Production/Stable',
 		'Programming Language :: Python',
diff --git a/test/segments_verify.py b/test/segments_verify.py
index c699d14..50f736f 100644
--- a/test/segments_verify.py
+++ b/test/segments_verify.py
@@ -527,25 +527,3 @@ if __name__ == "__main__":
 	failures = doctest.testmod(segments)[0]
 	if failures:
 		sys.exit(bool(failures))
-
-	# then with C extension implementation
-
-	from ligo.segments import __segments
-	segments.infinity = __segments.infinity
-	segments.NegInfinity = __segments.NegInfinity
-	segments.PosInfinity = __segments.PosInfinity
-	segments.segment = __segments.segment
-	segments.segmentlist = __segments.segmentlist
-
-	suite = unittest.TestSuite()
-	suite.addTest(unittest.makeSuite(test_infinity))
-	suite.addTest(unittest.makeSuite(test_segment))
-	suite.addTest(unittest.makeSuite(test_segmentlist))
-	suite.addTest(unittest.makeSuite(test_segmentlistdict))
-
-	if not unittest.TextTestRunner(verbosity=2).run(suite).wasSuccessful():
-		sys.exit(1)
-
-	failures = doctest.testmod(segments)[0]
-	if failures:
-		sys.exit(bool(failures))
